python/tests/compiler/control.py

```diff
@@ -24,11 +24,17 @@ def test_kernel_control_no_args(qubit_count):
     of qubits as the controls.
     """
     other_kernel = cudaq.make_kernel()
-    other_qubit = other_kernel.qalloc(qubit_count)
+    if qubit_count == 1:
+        other_qubit = other_kernel.qalloc()
+    else:
+        other_qubit = other_kernel.qalloc(qubit_count)
```

Comment:
This is very repetitive. Can't we make `kernel.qalloc(1)` just do the right thing?

Reply:
End of the day, and I was trying to minimally affect the existing FileCheck comments... 😞 I'll clean this up tomorrow

Reply:
Fixed.